client/me/get-apps/download-card.jsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Button from 'components/button';
+
+const DownloadCard = ( { translate } ) => {
+	const trackAppDownloadClick = () => {};
+	const availableLinkTag = ( platformLink ) => {
+		return <a href={ platformLink } onClick={ trackAppDownloadClick } />;
+	};
+
+	const windowsLink = 'https://apps.wordpress.com/d/windows?ref=getapps';
+	const macLink = 'https://apps.wordpress.com/d/osx?ref=getapps';
+	const linuxTarLink = 'https://apps.wordpress.com/d/linux?ref=getapps';
+	const linuxDebLink = 'https://apps.wordpress.com/d/linux-deb?ref=getapps';
+
+	let buttonLink = windowsLink;
+	let cardTitle = translate( 'Desktop App for Windows' );
+	let requires = translate( 'Requires Windows Whatevs v12+. ' );
+	let translateComponents = {
+		firstAvailableLink: availableLinkTag( macLink ),
+		secondAvailableLink: availableLinkTag( linuxTarLink ),
+		thirdAvailableLink: availableLinkTag( linuxDebLink ),
+	};
+	let alsoAvailable = translate( 'Also available for: {{firstAvailableLink}}MacOS{{/firstAvailableLink}},' +
+		'{{secondAvailableLink}}Linux (.tar.gz){{/secondAvailableLink}},' +
+		'{{thirdAvailableLink}}Linux (.deb){{/thirdAvailableLink}}.',
+		{ components: translateComponents }
+	);
+
+	if ( navigator.platform && navigator.platform.length > 0 ) {
```

Comment:
Done in 7742a791948a017cc19c48d3f4cefdb761b9731f and b55c056ea9acbfcd3d3d68ed65cc7802a1e3eb83.